src/Libraries/Microsoft.Extensions.AI/Functions/AIFunctionFactory.cs

```diff
@@ -46,35 +40,23 @@ public static AIFunction Create(Delegate method, AIFunctionFactoryCreateOptions
     /// <param name="method">The method to be represented via the created <see cref="AIFunction"/>.</param>
     /// <param name="name">The name to use for the <see cref="AIFunction"/>.</param>
     /// <param name="description">The description to use for the <see cref="AIFunction"/>.</param>
+    /// <param name="serializerOptions">The <see cref="JsonSerializerOptions"/> used to marshal function parameters.</param>
     /// <returns>The created <see cref="AIFunction"/> for invoking <paramref name="method"/>.</returns>
-    public static AIFunction Create(Delegate method, string? name, string? description = null)
-        => Create(method, (_defaultOptions ??= new()).SerializerOptions, name, description);
-
-    /// <summary>Creates an <see cref="AIFunction"/> instance for a method, specified via a delegate.</summary>
-    /// <param name="method">The method to be represented via the created <see cref="AIFunction"/>.</param>
-    /// <param name="options">The <see cref="JsonSerializerOptions"/> used to marshal function parameters.</param>
-    /// <param name="name">The name to use for the <see cref="AIFunction"/>.</param>
-    /// <param name="description">The description to use for the <see cref="AIFunction"/>.</param>
-    /// <returns>The created <see cref="AIFunction"/> for invoking <paramref name="method"/>.</returns>
-    public static AIFunction Create(Delegate method, JsonSerializerOptions options, string? name = null, string? description = null)
+    public static AIFunction Create(Delegate method, string? name = null, string? description = null, JsonSerializerOptions? serializerOptions = null)
```

Comment:
I don't believe it would be more common than before. CoreCLR apps wouldn't need to pass anything unless looking to customize things. I don't see a strong reason for removing it from the containing options class provided that we have relevant accelerator methods.

Reply:
> I don't believe it would be more common than before. CoreCLR apps wouldn't need to pass anything unless looking to customize things. I don't see a strong reason for removing it from the containing options class provided that we have relevant accelerator methods.

I wasn't suggesting removing it from AIFunctionFactoryCreateOptions. I was suggesting removing the `serializerOptions` parameter from this Create method.

Reply:
Why? this seems like a useful enough accelerator method.

Reply:
Sure, but can't the same be said about the `name` and `description` parameters? Point being, maybe we don't need an accelerator method at all?

Reply:
FWIW we have a number of tests that do customize by passing a JSO directly, if that's a measure of usefulness.

Reply:
Any scenario involving naming policies or passing in custom converters would require this. I don't think that accounts for the sub 1% use case. In any case, I don't think there's cost to adding this as another optional parameter assuming we keep the accelerator.

Reply:
> I don't think there's cost to adding this as another optional parameter assuming we keep the accelerator.

The cost is that:
a) It's more cognitive load when looking at the IntelliSense.
b) It puts the existence of JSO front and center when looking at IntelliSense.
c) It possibly complicates how future Create overloads are added.

But, like I said, if you believe it's really valuable, it's ok to keep it.

Reply:
I think it's an important enough component to building the function and its signature to put front and center.